book/website/communication/presentations/public-speaking-tips.md

```diff
@@ -0,0 +1,91 @@
+# Practical Techniques for Effective Public Speaking (or "A Guide to Impactful Presentations")
+
+Public speaking, like most other skills, is one you can never truly master, but only strive to refine. 
+
+Below are some practical tips that can radically sharpen your public speaking skills. 
+Whether you’re prepping for a talk, a webinar, or even a casual presentation, these gems might just transform how you approach your next speaking opportunity.
+
+## Resonance and Relevance
+At the core of every talk are two primary goals: to connect emotionally (resonance) and to deliver intellectually (relevance). 
+Striking the right balance ensures you’re not just heard, but truly remembered.
```

Comment:
I see your point. I think it can be even better if we avoid the word "balance" here and throughout the chapter. 

In my view, the word "balance" implies: 

1. Binary thinking where there are only two factors at play; 
2. That to have more of one, you necessarily lose an equal amount of the other; 
3. Subtly constrains our imagination and prevents win-win solutions. 

Therefore, I suggest this sentence be re-worded to "Managing both goals so that you're not just hard, but truly remembered". 

Same suggestion applies to other uses of "balance" in the text.

Reply:
Thanks for taking the time to provide this detailed explanation. I do understand and appreciate your perspective.

In this context, I chose the word "balance" to convey the idea of harmoniously integrating both emotional connection and intellectual delivery in public speaking. My intention is to emphasise the importance of both elements **working together** to create a memorable experience for the audience.

I hope this clarifies my decision to retain it in the text.